docs/extend/plugins.md

```diff
@@ -1,87 +1,272 @@
-[VMware vSphere Storage Plugin](https://github.com/vmware/docker-volume-vsphere)    | Docker Volume Driver for vSphere enables customers to address persistent storage requirements for Docker containers in vSphere environments.
+    ```bash
+    $ docker plugin install vieux/sshfs
 
-### Authorization plugins
+    Plugin "vieux/sshfs" is requesting the following privileges:
+    - network: [host]
+    - capabilities: [CAP_SYS_ADMIN]
+    Do you grant the above permissions? [y/N] y
 
- Plugin                                                       | Description                                                                                                                                                               
-------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- [Twistlock AuthZ Broker](https://github.com/twistlock/authz) | A basic extendable authorization plugin that runs directly on the host or inside a container. This plugin allows you to define user policies that it evaluates during authorization. Basic authorization is provided if Docker daemon is started with the --tlsverify flag (username is extracted from the certificate common name).
+    vieux/sshfs
+    ```
 
-## Troubleshooting a plugin
+    The plugin requests 2 privileges, the `CAP_SYS_ADMIN` capability to be able
+    to do mount inside the plugin and `host networking`.
 
-If you are having problems with Docker after loading a plugin, ask the authors
-of the plugin for help. The Docker team may not be able to assist you.
+2. Check for a value of `true` the `ENABLED` column to verify the plugin
+started without error.
 
-## Writing a plugin
+    ```bash
+    $ docker plugin ls
 
-If you are interested in writing a plugin for Docker, or seeing how they work
-under the hood, see the [docker plugins reference](plugin_api.md).
+    NAME                TAG                 ENABLED
+    vieux/sshfs         latest              true
+    ```
+
+3. Create a volume using the plugin.
+
+    ```bash
+    $ docker volume create \
+      -d vieux/sshfs \
+      --name sshvolume \
+      -o sshcmd=user@1.2.3.4:/remote
+
+    sshvolume
+    ```
+
+4.  Use the volume `sshvolume`.
+
+    ```bash
+    $ docker run -v sshvolume:/data busybox ls /data
+
+    <content of /remote on machine 1.2.3.4>
+    ```
+
+5. Verify the plugin successfully crated the volume.
+
+    ```bash
+    $ docker volume ls
+
+    DRIVER              NAME
+    vieux/sshfs         sshvolume
+    ```
+
+    You can stop a plugin with the `docker plugin disable`
+    command or remove a plugin with `docker plugin remove`.
+
+See the [command line reference](../engine/reference/commandline/) for more
```

Comment:
Also, please link the plugins API reference doc Section 3.6 at https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.25.md